src/app/digital/views/index.ts

```diff
@@ -22,6 +24,7 @@ export const Views: ViewRecord<DigitalObj, DigitalViewInfo> = {
     "Switch":  (c, o) => new SwitchView(c, o),
     "LED":     (c, o) => new LEDView(c, o),
     "ANDGate": (c, o) => new ANDGateView(c, o),
+    "NOTGate": (c, o) => new ComponentView<NOTGate, DigitalViewInfo>(c, o, V(1, 1), "buf.svg"),
```

Comment:
nit: you can leave out the template arguments since they get automatically inferred